src/core/templates/ResourceMembers.ts

```diff
@@ -6,15 +6,21 @@ import {
   RequestHelper,
   Sudo,
 } from '../infrastructure';
-
 import { AccessLevel } from './ResourceAccessRequests';
 
+interface IncludeInherited {
+  includeInherited?: boolean;
```

Comment:
hmmm

Argument of type '{ includeInherited: boolean; }' is not assignable to parameter of type 'boolean'.ts(2345)

`api.ProjectMembers.all(project.id, { includeInherited: false })`